icu4j/build.properties

```diff
@@ -6,7 +6,7 @@
 #*******************************************************************************
 api.report.version = 64
 api.report.prev.version = 63
-release.file.ver = 64_1
-api.doc.version = 64.1
-maven.pom.ver = 64.1
+release.file.ver = 64_2
```

Comment:
Wasn't sure if this should be "64rc" or "64_2rc" instead? Docs state this:

> release.file.ver: For release candidate, use <2-digit major version> + "rc", e.g. 54rc.

Reply:
That's a good question. :)
I don't think there is any plan to do two "turns of the crank" for 64.2, given that there is only one column in the ICU 64 BRS spreadsheet. (We'd need to have another column for 64.2 GA to then remove the "rc" bit). So I think just having this bump up to 64.2 is fine.

Note that this is also how things were handled for the ICU 60.2 release as well.